platform_usermanagement/index.js

```diff
@@ -309,4 +330,23 @@ function createUserInBitBucket(config, userData) {
 		},
 		qs: {}
 	};
-}
\ No newline at end of file
+}
+
+function createUserInSCM(config, userData) {
+  //bitbucket implementation
+  if(config.scm_service_host.includes("bitbucket")){
+  	createUserInBitBucket(config, userData);
+  }
+  //Gitlab implementation
+  else if(config.scm_service_host.includes("gitlab")){
+    createUserInGitlab(config, userData);
+  }
+  else if (config.scm_service_host){
+    logger.error("Currently, the scm: " + config.scm_service_host + " is not supported");
+    return reject(errorHandler.throwInputValidationError("110", "scm not supported"));
+  }
+  else{
+    logger.error("scm_service_host is not defined");
```

Comment:
This condition of "scm_service_host is not defined" will never be hit. We will hit it in the first _if_ where it will fail as we are calling includes on undefined. 

We need this check though, so how about you move it to input validation (validateCreaterUserParams).

Reply:
validation check has been moved